lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -172,6 +175,33 @@ private function persistGeneric(
     }
 
     /**
+     * Persist "json_array" field types.
+     *
+     * @param NodeInterface $node
+     * @param DocumentAccessor $accessor
+     * @param mixed $fieldName
+     * @param mixed $locale
+     * @param array $fieldMapping
+     */
+    private function persistJsonArray(
+        NodeInterface $node,
+        DocumentAccessor $accessor,
+        $fieldName,
+        $locale,
+        array $fieldMapping
+    ) {
+        try {
+            $phpcrName = $this->encoder->encode($fieldMapping['encoding'], $fieldMapping['property'], $locale);
+            $value = $accessor->get($fieldName);
+            $this->validateFieldValue($value, $fieldName, $fieldMapping);
+            $node->setProperty($phpcrName, json_encode($value));
+        } catch (InvalidLocaleException $ex) {
+            // arguments unvalid no valid propertyname could be generated (e.g. no locale given for localized encoding)
```

Comment:
but this is done for all "types" in this class maybe add an issue? typo is fixed